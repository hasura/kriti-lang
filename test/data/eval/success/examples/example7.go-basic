[
  {{ range _, x := $ }}
    {{ range i, friend := x.friends }}
      {"name": {{ friend.name }}}
    {{ end }}
  {{ end }}
]