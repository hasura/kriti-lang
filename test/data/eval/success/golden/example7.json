[
    [
        [
            {
                "name": "Wilkerson Gill"
            },
            {
                "name": "Gonzalez Long"
            },
            {
                "name": "Hanson Ellison"
            }
        ],
        [
            {
                "name": "Shanna Emerson"
            },
            {
                "name": "Charmaine Tate"
            },
            {
                "name": "Elisa Albert"
            }
        ],
        [
            {
                "name": "Ford Crosby"
            },
            {
                "name": "Owens Vaughn"
            },
            {
                "name": "Lesley Lopez"
            }
        ],
        [
            {
                "name": "Megan Hahn"
            },
            {
                "name": "Green Mathews"
            },
            {
                "name": "Maddox Tucker"
            }
        ],
        [
            {
                "name": "Patton Larson"
            },
            {
                "name": "Casey Grimes"
            },
            {
                "name": "Geneva Carroll"
            }
        ]
    ]
]